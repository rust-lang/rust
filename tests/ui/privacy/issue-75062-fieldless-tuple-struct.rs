// Regression test for issue #75062
// Tests that we don't ICE on a privacy error for a fieldless tuple struct.

mod foo {
    struct Bar();
}

fn main() {
    foo::Bar(); //~ ERROR tuple struct
}
