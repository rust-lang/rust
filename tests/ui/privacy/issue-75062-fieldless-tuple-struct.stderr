error[E0603]: tuple struct `Bar` is private
  --> $DIR/issue-75062-fieldless-tuple-struct.rs:9:10
   |
LL |     foo::Bar();
   |          ^^^ private tuple struct
   |
note: the tuple struct `Bar` is defined here
  --> $DIR/issue-75062-fieldless-tuple-struct.rs:5:5
   |
LL |     struct Bar();
   |     ^^^^^^^^^^^^^

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0603`.
